src/app/components/faq/FaqQuestion/faqQuestion.js

```diff
@@ -0,0 +1,21 @@
+
+
+import { useState } from 'react'
+
+export default function FaqQuestion(props) {
+    let [expanded, setExpanded] = useState(false)
+
+    function toggle() {
+        setExpanded(!expanded);
+    }
+
+    return (
+        /*button borders*/
+        <div className="border border-sky-500 rounded-md mb-4">
+            <button className="w-full bg-sky-800 text-white px-2 py-2 md:px-4 md:py-2 rounded-md font-sans text-sm md:text-lg border-b border-sky-500 cursor-pointer hover:opacity-80" onClick={toggle}>{props.question}</button>
```

Comment:
Just looked at the text size on my phone, it seems kind of small and makes the buttons hard to click on. Consider adding more padding and making it text-base or lg instead of text-sm by default

Reply:
done - changed to text-base, increased padding slightly (though it seems fine to me?)